src/components/FormSections/PersonalInfo.jsx

```diff
@@ -323,40 +356,46 @@ function PersonalInfo(props){
                             </label>
                         </div>
                     </div>
-                </Fieldset>
             {(dobReq && handleErrors.dob) && 
                 <span id="dob-error" rol="alert" className='error-text'>
                     Date of Birth must follow the format of January 19 2000.
                 </span>
             }
             </div>
+        </Grid>
         )}
+            
 
+            
         {telephoneVisible && (
-            <div className={(telephoneReq && handleErrors.phone_number) ? 'error-container' : ''}>
-                <legend htmlFor="phone-number" className="usa-legend">Phone Number{telephoneReq && <span className='required-text'>*</span>}</legend>
-                <span className="usa-hint" id="date-of-birth-hint">For example: (123) 456-7890</span>
-                <TextInput 
-                    id="phone-number" 
-                    aria-describedby="phone-number-error"
-                    name="phone-number" 
-                    type="text" 
-                    autoComplete="off" 
-                    required={telephoneReq}
-                    maxLength={14}
-                    minLength={14}
-                    value={props.fieldData.phone_number} 
-                    onChange={props.saveFieldData('phone_number')} 
-                    onBlur={(e) => setHandleErrors({ ...handleErrors, phone_number: checkForErrors(e, 'check value length') })}
-                />
-                {(telephoneReq && handleErrors.phone_number) && 
-                    <span id="phone-number-error" rol="alert" className='error-text'>
-                        Phone number must be 10 digits.
-                    </span>
-                }
-            </div>
+            <Grid tablet={{ col: true }} className="input-example">
+                <div className={(telephoneReq && handleErrors.phone_number) ? 'error-container bottom' : 'bottom'}>
+                    <legend htmlFor="phone-number" className="usa-legend">Phone Number{telephoneReq && <span className='required-text'>*</span>}</legend>
```

Comment:
orphaned legend

Reply:
I checked with Amy and since phone number is a single input and not a group it is okay to leave out fieldset here